Celeste.Mod.mm/Patches/GameLoader.cs

```diff
@@ -168,6 +168,14 @@ class patch_GameLoader : GameLoader {
                 previousVersion = new Version(0, 0, 0);
             }
 
+#if DEBUG
+            // skip creating a backup if we are on a develop build
+            if (previousVersion == new Version(0, 0, 0)) {
+                Logger.Log(LogLevel.Verbose, "core", "Running in debug build, skipping backup");
```

Comment:
```suggestion
                Logger.Log(LogLevel.Verbose, "core", "Running in development build, skipping backup");
```

Reply:
Sure makes sense, I have committed the suggested changes.